src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -1465,12 +1465,8 @@ function Test-ApplicationGatewayCRUDSubItems2
 		# Add to test Remove
 		Add-AzApplicationGatewayBackendHttpSettings -ApplicationGateway $appgw -Name $poolSetting02Name -Port 1234 -Protocol Http -CookieBasedAffinity Enabled -RequestTimeout 42 -HostName test -Path /test -AffinityCookieName test
 		$fipconfig = Get-AzApplicationGatewayFrontendIPConfig -ApplicationGateway $appgw -Name $fipconfigName
-		$fp02 = Get-AzApplicationGatewayFrontendPort -ApplicationGateway $appgw -Name $frontendPort02Name 
```

Comment:
@EvgenyAgafonchikov just to clarify, is this a scenario that no longer works?

Reply:
@cormacpayne, this scenario should still work.

Previous approach dropped IDs from all the subresources (e.g. FrontendPort) ApplicationGateway was created with, so we had to re-request them to refer again. After updates in this PR IDs are kept, so re-requesting is no more required. This also repros scenario from the related issue #3800